cds-snc/cppd-medical-report-sails
# People know what to do if they forget their invitation code

# What this does

Updates the Medical Professional login screen according to the [new design described here](https://trello.com/c/7gGi0Aj1/499-people-know-what-to-do-if-they-forget-their-invitation-code). Note that there is some related work in #137 

## To test
- Navigate to the Medical Professional login `/en/medical-professional`
- See the new date field
- See the new message at the bottom

## Note for SWE code reviewers
This PR adds a config entry and environment variable for baseURL. The intention is to set this based on environment in config/environment. Note that in the InviteController we handle this a bit differently, using various `req.` properties to build up a base url. 

[According to the Sails documentation](https://sailsjs.com/documentation/reference/application/advanced-usage/sails-get-base-url), "There is no reliable, cross-platform way to automatically detect the external URL of a running Sails app (or any other Node app). Instead, configure your base URL explicitly and save it in custom configuration."

So recommend going back and retrofitting the InviteController (and any other circumstances where we've used this method to build up a baseURL).
